### ISSUE TITLE
Has development stopped?

### ISSUE THREAD

**Issue by EpicLPer** [maruos/maruos#95]
There was no commit or release in over a year and in general the devs seem to be somewhat gone... Has this project come to an end? Would be sad to see :(

**Comment by pdsouza** [comment]
Lots of stuff happening!

This repo is just a landing page for the project where we host issues--there is no code here. Please check out individual project repos under the [maruos org](https://github.com/maruos) to assess project activity. You'll see that we've had quite a bit of activity on our new `maru-0.6` branch (I literally [just merged a PR](https://github.com/maruos/vendor_maruos/commit/feb23fbd90c4186fff7b6e5c0bb214ddd2e2766b) this morning). 

We also don't use GitHub that much since most of our core community uses our [dev mailing list](https://groups.google.com/forum/#!forum/maru-os-dev). I highly recommend subscribing to our mailing list if you would like to keep up with the latest.

I agree we haven't had an official release in a while; some of the reasons why is outlined in a [blog post](https://blog.maruos.com/2017/06/29/a-year-of-maru/) I wrote. But development has kicked up recently, and we have a new release coming out soon that will be based on Android 8.1 / Oreo. There has also been a lot of work that has been going on the past few months to automate our releases so we can get them out more regularly, see the releases section under our [builds](https://github.com/maruos/builds/releases) project that just came online last week. The first release for Maru 0.6 is already listed there and is being tested internally. Once things are stable they will be released publicly.

**tl;dr** We had some changes over the past year that slowed down development for a while but we are very much alive and kicking! Look forward to Maru on Android 8.1 with expanded device support and regular releases this year.

**Comment by EpicLPer** [comment]
Awesome! :)
Now I'll wait for someone of the community to port it over to the OnePlus 3/5/6 since I own those haha, the Nexus 5 is a bit dated already :)

**Comment by pdsouza** [comment]
Closing. Feel free to open another issue if you have any other questions.
